website_product_subscription/controllers/main.py

```diff
@@ -79,6 +79,7 @@ def get_receiver(self, kwargs):
         vals['name'] =  firstname + ' ' + lastname
         vals['firstname'] = firstname
         vals['lastname'] = lastname
+        subscriber_vals["customer"] = True

```

Comment:
subscriber_vals doesn't exist in this function please change it for vals as the preceding lines